contracts/test/accounts/Governor.sol

```diff
@@ -13,157 +12,33 @@ contract Governor {
-        (ok,) = globals.call(abi.encodeWithSignature(sig, defaultGracePeriod));
-    }
-
-    function try_setMinLoanEquity(address globals, uint256 newLiquidity) external returns (bool ok) {
-        string memory sig = "setMinLoanEquity(uint256)";
-        (ok,) = globals.call(abi.encodeWithSignature(sig, newLiquidity));
-    }
-
-    function try_setFundingPeriod(address globals, uint256 fundingPeriod) external returns (bool ok) {
-        string memory sig = "setFundingPeriod(uint256)";
-        (ok,) = globals.call(abi.encodeWithSignature(sig, fundingPeriod));
-    }
-
-    function try_setSwapOutRequired(address globals, uint256 swapAmt) external returns (bool ok) {
-        string memory sig = "setSwapOutRequired(uint256)";
-        (ok,) = globals.call(abi.encodeWithSignature(sig, swapAmt));
-    }
-
-    function try_setPriceOracle(address globals, address asset, address oracle) external returns (bool ok) {
-        string memory sig = "setPriceOracle(address,address)";
-        (ok,) = globals.call(abi.encodeWithSignature(sig, asset, oracle));
-    }
-
-    function try_setPendingGovernor(address globals, address pendingGov) external returns (bool ok) {
-        string memory sig = "setPendingGovernor(address)";
-        (ok,) = globals.call(abi.encodeWithSignature(sig, pendingGov));
-    }
-
-    function try_acceptGovernor(address globals) external returns (bool ok) {
-        string memory sig = "acceptGovernor()";
-        (ok,) = globals.call(abi.encodeWithSignature(sig));
-    }
-
-    function try_setGlobals(address target, address _globals) external returns (bool ok) {
+    /*********************/
+    /*** Try Functions ***/
+    /*********************/
+    function try_treasury_setGlobals(address target, address _globals) external returns (bool ok) {
         string memory sig = "setGlobals(address)";
         (ok,) = address(target).call(abi.encodeWithSignature(sig, _globals));
     }
 
-    function try_reclaimERC20_treasury(address treasury, address asset, uint256 amount) external returns (bool ok) {
+    function try_treasury_reclaimERC20(address treasury, address asset, uint256 amount) external returns (bool ok) {
         string memory sig = "reclaimERC20(address,uint256)";
         (ok,) = address(treasury).call(abi.encodeWithSignature(sig, asset, amount));
     }
 
-    function try_distributeToHolders(address treasury) external returns (bool ok) {
+    function try_treasury_distributeToHolders(address treasury) external returns (bool ok) {
         string memory sig = "distributeToHolders()";
         (ok,) = address(treasury).call(abi.encodeWithSignature(sig));
     }
 
-    function try_convertERC20(address treasury, address asset) external returns (bool ok) {
+    function try_treasury_convertERC20(address treasury, address asset) external returns (bool ok) {
         string memory sig = "convertERC20(address)";
         (ok,) = address(treasury).call(abi.encodeWithSignature(sig, asset));
     }
```

Comment:
Don't need to cast any of these to `address`, can just do 
`(ok,) = treasury.call(abi.encodeWithSignature(...));`